### PR TITLE
Add missing 'active products' translation on dashboard

### PR DESCRIPTION
**Pull request by haseleyi**
#### What? Why?

Closes [#3290]

The admin dashboard is missing a translation for "You have XX active products".

#### Release notes

Added missing translation for "You have XX active products" in admin dashboard.

Changelog Category: Fixed

### CODE REVIEW

**Comment by haseleyi** [comment]
Love it! Thanks Luis.

**Comment by luisramos0** [comment]
@mkllnk that pluralization thing is amazing! thanks for the lesson.

**Comment by haseleyi** [comment]
Thanks a ton for this @mkllnk! Here is the one/other change. Regarding the zero-product refactor, how can we remove the if/else but still have the .red and new-product pieces conditional on having zero products?

File: app/views/spree/admin/overview/_products.html.haml [hunk 1]
```diff
@@ -12,10 +12,7 @@
     - if @product_count > 0
       %div.seven.columns.alpha.list-item
         %span.six.columns.alpha
-          - if @product_count == 1
-            = t(".active_products_count_single", product_count: @product_count )
-          - else
-            = t(".active_products_count_multiple", product_count: @product_count )
+          = t(".active_products", count: @product_count )
```
Comment (mkllnk) [inline app/views/spree/admin/overview/_products.html.haml]: You can use exactly the same code in line 19 and move the translation from `spree_admin_enterprises_producers_manage_products` to `spree.admin.overview.products.active_products.zero`.

**Comment by mkllnk** [comment]
I looked at it again and it's not worth it to remove the if-else block because you would need four nested if-else operators for the right content. I thought about this:
```diff
-%div.seven.columns.alpha.list-item
+%div.seven.columns.alpha.list-item{class: "red" if @product_count == 0}
```

If it was just that one class, it would be good, but there are too many. I just commented on one little thing you could improve, but it's good as it is.

**Comment by luisramos0** [comment]
no, the build is fine. we are taking care of those failing tests in other PRs, see "flaky spec" kind of issue/PR.

**Comment by RachL** [comment]
All related text looks normal:
image

**Comment by sauloperez** [comment]
I just merged `master` into your branch @haseleyi so that we get all the fixed specs to see the build passing. This way I don't mess with your branch's history as a rebase would do.

**Comment by sigmundpetersen** [comment]
@sauloperez this will need another merge from master with the broken product import spec fix

**Comment by luisramos0** [comment]
yeah, correct. but that broken spec has nothing to do with this PR. I'd just merge this PR.

**Comment by sauloperez** [comment]
Ok, doing an exception here to avoid @haseleyi having to deal with it but I really dislike merging red PRs. I feel like we create a dangerous precedent.

**Pull request merged**